cloud_functions/token_refresh/test_main.py

```
@@ -0,0 +1,128 @@
+from .main import refresh_token
+
+
+def create_test_event(user_id: str) -> dict:
+    """Create a test Cloud Event."""
+    return {
+        "data": {"message": {"data": json.dumps({"user_id": user_id}).encode("utf-8")}}
+    }
+
+
+@pytest.fixture
+def mock_strava_response():
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_strava_response`

(ANN201)